src/signable/hashes.ts

```diff
@@ -57,6 +54,6 @@ export async function preComputeHashes(): Promise<void> {
     })),
 
     // Conditional transfers: hash(transfer asset, fee asset)
-    getCacheablePedersenHash(collateralAssetBn, collateralAssetBn),
+    getCacheablePedersenHash(collateralAssetBn, CONDITIONAL_TRANSFER_FEE_ASSET_ID_BN),
```

Comment:
I don't think this fix is related -- I'm trying to see why the test case in `hashes.test.ts` didn't fail earlier

Reply:
Ah it's because `0` used to be one of our synthetic asset IDs